nixos/modules/programs/ssh.nix

```diff
@@ -84,6 +84,21 @@ in
         '';
       };
 
+
+      agentPkcs11Whitelist = mkOption {
```

Comment:
Could it be named `agentPkcs11Modules` or are those not really loaded with this options?

Reply:
They are not loaded. They're only whitelisted.  One always has to manually specify the path when invoking `ssh-add` unfortunately. See the linked OpenSSH issue.

Reply:
Ok. Just read the discussion above.

Reply:
It would be nice to add an usage example you have in your description above also in the module description.

Reply:
Sounds like a good idea. Just hadn't bothered yet because docbook confuses me :P I'll have to read up on docbook first before figuring out how to format it correctly